src/System.Security.Claims/src/System.Security.Claims.builds

```diff
@@ -5,6 +5,12 @@
     <Project Include="facade\System.Security.Claims.csproj" Condition="'$(TargetsWindows)' == 'true'">
       <AdditionalProperties>TargetGroup=net46</AdditionalProperties>
     </Project>
+    <Project Include="System.Security.Claims.csproj">
+      <AdditionalProperties>TargetGroup=</AdditionalProperties>
+    </Project>
+    <Project Include="System.Security.Claims.csproj">
+      <AdditionalProperties>TargetGroup=net46</AdditionalProperties>
```

Comment:
This is a duplication of facade\System.Secrit.Claims.csproj configuration. As part of your change you should eliminate the facade folder completely given you have folded it into this project.  Also condition this on TargetsWindows=true